static/js/App.tsx

```diff
@@ -25,39 +25,39 @@ const App: FC = () => {
         element={<ProtectedRoute outlet={<InstanceList />} />}
       />
       <Route
-        path="/ui/instances"
+        path="/ui/:project/instances"
         element={<ProtectedRoute outlet={<InstanceList />} />}
       />
       <Route
-        path="/ui/instances/:name"
+        path="/ui/:project/instances/:name"
         element={<ProtectedRoute outlet={<InstanceDetail />} />}
       />
       <Route
-        path="/ui/instances/:name/:activeTab"
+        path="/ui/:project/instances/:name/:activeTab"
         element={<ProtectedRoute outlet={<InstanceDetail />} />}
       />
       <Route
         path="/ui/images"
         element={<ProtectedRoute outlet={<ImageList />} />}
       />
       <Route
-        path="/ui/profiles"
+        path="/ui/:project/profiles"
         element={<ProtectedRoute outlet={<ProfileList />} />}
       />
       <Route
-        path="/ui/profiles/:name"
+        path="/ui/:project/profiles/:name"
         element={<ProtectedRoute outlet={<ProfileDetail />} />}
       />
       <Route
-        path="/ui/networks"
+        path="/ui/:project/networks"
         element={<ProtectedRoute outlet={<NetworkList />} />}
```

Comment:
The networks api file and all the points where `fetchNetworks` gets referenced (`NetworkList` and `NetworkSelector`) need to be updated too.